tests/test_query.py

```diff
@@ -551,6 +553,22 @@ def test_query_order_by_random(self):
 
         assert list(qs) != list(qs.order_by('?'))
```

Comment:
Hey @stphivos,
This is a flaky test because sometimes the random order will be the same as the initial order.
It's what broke the latest build for this PR.

Reply:
Thx, you're right https://github.com/stphivos/django-mock-queries/pull/145 is an attempt to fix that